<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SizeSetting</class>
 <widget class="QDialog" name="SizeSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>188</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>409</width>
    <height>188</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>409</width>
    <height>188</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Size Setting</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>381</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>381</width>
     <height>20</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_I">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>192</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>192</width>
     <height>81</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>192</width>
     <height>81</height>
    </size>
   </property>
   <property name="title">
    <string>Input Place</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Row</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="plainTextEdit_IC"/>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit_IR"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Column</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_O">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>192</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>192</width>
     <height>81</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>192</width>
     <height>81</height>
    </size>
   </property>
   <property name="title">
    <string>Output Place</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Row</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="plainTextEdit_OC">
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit_OR"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Column</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_TC">
   <property name="geometry">
    <rect>
     <x>208</x>
     <y>10</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_TC">
   <property name="geometry">
    <rect>
     <x>208</x>
     <y>28</y>
     <width>190</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_TR">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>28</y>
     <width>190</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_TR">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>102</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SizeSetting</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SizeSetting</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
